app/services/audit_results.rb

```diff
@@ -97,12 +97,12 @@ class AuditResults
   ].freeze
```

Comment:
and would instead suggest commenting the replication related result codes in this list, similar to what's done w/ the HB codes.

alternatively, we could just comment out `results.report_results(logger)` (as you've done above), and leave the reporting codes in this class alone entirely.  i think i have a slight preference for commenting the result codes... but mostly i'd want to just comment result codes, or just comment the reporting invocation, but not half-n-half.